charts/opentelemetry-demo/values.yaml

```diff
@@ -500,14 +516,1674 @@ opentelemetry-collector:
+      otlp/spanmetrics:
+        protocols:
+          grpc:
+            endpoint: "localhost:65535"
+      prometheus:
+        config:
+          scrape_configs:
+            - job_name: 'otel-collector'
+              scrape_interval: 5s
+              static_configs:
+                - targets: ['0.0.0.0:8888']
+
     exporters:
       otlp:
-        endpoint: '{{ .Release.Name }}-jaeger:4317'
+        endpoint: '{{ include "otel-demo.name" . }}-jaeger:4317'
         tls:
           insecure: true
+      logging:
+      prometheus:
+        endpoint: '0.0.0.0:9464'
+
+    processors:
+      batch:
+      spanmetrics:
+        metrics_exporter: prometheus
+
     service:
+      telemetry:
+        metrics:
+          level: detailed
+          address: 0.0.0.0:8888
       pipelines:
         traces:
-          exporters:
-            - logging
-            - otlp
+          receivers: [otlp]
+          processors: [spanmetrics, batch]
+          exporters: [otlp, logging]
+        metrics:
+          receivers: [otlp, prometheus]
+          processors: [batch]
+          exporters: [prometheus, logging]
+        metrics/spanmetrics:
+          receivers: [otlp/spanmetrics]
+          exporters: [prometheus, logging]
+
+prometheus:
+  alertmanager:
+    enabled: false
+  configmapReload:
+    prometheus:
+      enabled: false
+  kubeStateMetrics:
+    enabled: false
+  nodeExporter:
+    enabled: false
+  pushgateway:
+    enabled: false
+
+  server:
+    global:
+      scrape_interval: 5s
+      scrape_timeout: 3s
+      evaluation_interval: 30s
+    service:
+      servicePort: 9090
+
+  serverFiles:
+    prometheus.yml:
+      scrape_configs:
+        - job_name: 'opentelemetry-community-demo'
+          kubernetes_sd_configs:
+            - role: pod
+          relabel_configs:
+            - source_labels: [__meta_kubernetes_pod_annotation_opentelemetry_community_demo]
+              action: keep
+              regex: true
+
+grafana:
+  grafana.ini:
+    auth:
+      disable_login_form: true
+    auth.anonymous:
+      enabled: true
+      org_name: Main Org.
+      org_role: Admin
+  adminPassword: admin
+  datasources:
+    datasources.yaml:
+      apiVersion: 1
+      datasources:
+        - name: Prometheus
+          uid: webstore-metrics
+          type: prometheus
+          url: 'http://{{ include "otel-demo.name" . }}-prometheus-server:9090'
+          editable: true
+          isDefault: true
+        - name: Jaeger
+          uid: webstore-traces
+          type: jaeger
+          url: 'http://{{ include "otel-demo.name" . }}-jaeger:16686'
+          editable: true
+          isDefault: false
+  dashboardProviders:
+    dashboardproviders.yaml:
+      apiVersion: 1
+      providers:
+        - name: 'default'
+          orgId: 1
+          folder: ''
+          type: file
+          disableDeletion: false
+          editable: true
+          options:
+            path: /var/lib/grafana/dashboards/default
+  dashboards:
```

Comment:
I'm not psyched about this massive dashboard in the values.yaml.  Is there a way to provide it via its own file/template?

Reply:
I moved it to a configmap ... much cleaner this way